src/app/(app)/admin/maps/[id]/edit/index.tsx

```
@@ -1,24 +1,29 @@
 import Button from '@/components/button'
 import Dropdown from '@/components/dropdown'
 import Input from '@/components/input'
+import SelectLocation from '@/components/select-location'
 import useCities from '@/hooks/use-cities-instant'
+import { useThemedColors } from '@/hooks/use-themed-colors'
 import { Map } from '@/interfaces'
```

Biome (2.1.2)

[error] Do not shadow the global "Map" property.

Consider renaming this variable. It's easy to confuse the origin of variables when they're named after a known global.

(lint/suspicious/noShadowRestrictedNames)